cmd/cert-rotation/main.go

```diff
@@ -82,16 +90,29 @@ func realMain(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to setup kms client: %w", err)
 	}
-	defer cleanup.GracefulClose(logger, kmsClient)
+	defer util.GracefulClose(logger, kmsClient)
 
-	config, err := config.LoadCryptoConfig(ctx, []byte{})
+	cryptoCfg, err := config.LoadCryptoConfig(ctx, []byte{})
 	if err != nil {
 		return fmt.Errorf("failed to load config: %w", err)
 	}
 
+	firestoreDoc, err := util.ParseFirestoreDocResource(cryptoCfg.FirestoreDocResourceName)
```

Comment:
I don't know if we want this to be firestore per the doc just shared out. I'm worried about the coupling here.

Reply:
If we're unsure about firestore, maybe we should defer using firestore at all. It's not like we're missing functionality without it. Thoughts?

Reply:
Agree on deferring

Reply:
Ok. I will revert the previous PR which already uses firestore https://github.com/abcxyz/jvs/pull/91